apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/CharSequenceContainsAtMostAssertionsSpec.kt

```diff
@@ -26,23 +30,33 @@ class CharSequenceContainsAtMostAssertionsSpec : ch.tutteli.atrium.spec.integrat
             Companion::containsAtMost
         )
 
-        private fun containsAtMost(plant: Assert<CharSequence>, atMost: Int, a: Any, aX: Array<out Any>)
-            = plant to contain atMost atMost the Values(a, *aX)
+        private fun containsAtMost(plant: Assert<CharSequence>, atMost: Int, a: Any, aX: Array<out Any>): AssertionPlant<CharSequence> {
+            val values = Values(a, *aX)
+            return (plant.asExpect().contains(o) atMost atMost).the(
```

Comment:
`atMost` is still from cc-infix. I guess you did the replacement but the problem is, that we have a star import: `import ch.tutteli.atrium.api.infix.en_GB.*` and it seems like the one of cc-infix still takes precedence in this case. Hence add `import ch.tutteli.atrium.api.infix.en_GB.atMost` in addition (the replacement should have done that, that's a bug in kotlin)